src/blurredLocation.js

```diff
@@ -238,6 +238,7 @@ BlurredLocation = function BlurredLocation(options) {
     if(isBlurred()) {
         drawCenterRectangle(bounds);
         disableCenterMarker();
+        options.map.on('move',updateRectangleOnPan);
```

Comment:
hmm, would this not be circular since it occurs inside `updateRectangleOnPan()`? https://github.com/publiclab/leaflet-blurred-location/pull/114/files#diff-8914aa74753366419a4ebf41e78f06f3R234

Reply:
Oh right ! Everytime when blurred = true and we are panning the map , this will apply this event-listener again and again . It solves the problem but yes not the best solution . Will make changes . Thanks 👍 .